src/theory/strings/theory_strings.cpp

```diff
@@ -3464,6 +3486,15 @@ bool TheoryStrings::processLoop( std::vector< std::vector< Node > > &normal_form
   }
   else
   {
+    if (options::stringProcessLoopMode() == ProcessLoopMode::SIMPLE_ABORT)
+    {
+      throw LogicException("Normal loop breaking not allowed");
```

Comment:
I think this should say something like `Normal looping word equation encountered.`